apps/opt.c

```diff
@@ -260,10 +259,6 @@ int opt_format(const char *s, unsigned long flags, int *result)
             if ((flags & OPT_FMT_PEMDER) == 0)
                 return opt_format_error(s, flags);
             *result = FORMAT_PEM;
-        } else if (strcmp(s, "PVK") == 0 || strcmp(s, "pvk") == 0) {
-            if ((flags & OPT_FMT_PVK) == 0)
-                return opt_format_error(s, flags);
-            *result = FORMAT_PVK;
```

Comment:
Is it really a really a good idea to remove the pvk related command line option(s) without substitution? It might be better to leave them in place to provide a better error message when failing.

Reply:
You do get an error saying the format is unrecognized:
```
; ./apps/openssl dsa -inform xxx
dsa: Invalid format "xxx" for -inform
dsa: Use -help for summary.
```
And pasting that example pointed out some missing refs in the help messages.
Additional commit pushed.